### PR TITLE
fix(tabs): add classNames to tab

### LINKED ISSUE #4567: Adding className to Tab overrides default styling
**Describe the issue. What is the expected and unexpected behavior?**
When adding a `className` property `Tab` component, append it to initial className list

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**
Add className to `Tab` component
```jsx
<Tab className="foo" ... />
```
results in: 
screenshot-www patternfly org-2020 07 17-10_28_31

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug

**What is your product and what release version are you targeting?**
CRC. PF version: @patternfly/react-core@4.23.1

### PR DESCRIPTION
**Pull request by boaz0**
**What**: Closes #4567

### CODE REVIEW

File: packages/react-core/src/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -256,13 +256,19 @@ export class Tabs extends React.Component<TabsProps, TabsState> {
                   id: childId,
                   tabContentId,
                   isHidden = false,
+                  className: childClassName = '',
```
Comment (tlabaj) [inline packages/react-core/src/components/Tabs/Tabs.tsx:259]: Can you try just using classname here.  It looks like `className` was not destructured before and that might have been the issue.  I don't think you need to rename it to `childClassName`.

Comment (boaz0) [reply packages/react-core/src/components/Tabs/Tabs.tsx:259]: I was afraid of `className` in line 197 shadowing this `className`. But you are probably right.

Comment (boaz0) [reply packages/react-core/src/components/Tabs/Tabs.tsx]: Well, nope - it didn't work. eslint is not happy:

```
/home/circleci/project/packages/react-core/src/components/Tabs/Tabs.tsx
  259:19  error  'className' is already declared in the upper scope                                                                                                                                                                                                                      no-shadow
  266:36  error  Replace `⏎······················styles.tabsItem,⏎······················eventKey·===·activeKey·&&·styles.modifiers.current,⏎······················className⏎····················` with `styles.tabsItem,·eventKey·===·activeKey·&&·styles.modifiers.current,·className`  prettier/prettier
```

**Comment by boaz0** [comment]
Thank you all! :pray:

**Pull request merged**